pkg/nodeagent/node_agent.go

```diff
@@ -112,6 +112,10 @@ type Configs struct {
 
 	// PodResources is the resource config for various types of pods launched by node-agent, i.e., data mover pods.
 	PodResources *kube.PodResources `json:"podResources,omitempty"`
+
+	// PriorityClassName is the priority class name for both the node agent daemonset
```

Comment:
This is not true. The priority class name for node-agent daemonset is set by installation CLI.
This config is for data mover pods only.